ecs-cluster/iam.tf

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
@simonrondelez don't forget that you can link specific roles to your ECS containers. Not all roles & profiles need to end up on the instance.